kai/reactive_codeplanner/task_runner/dependency/task_runner.py

```diff
@@ -67,6 +68,9 @@ def execute_task(self, rcm: RepoContextManager, task: Task) -> TaskResult:
         maven_dep_response = self._agent.execute(MavenDependencyRequest(task.file, msg))
         logger.info("got mvn dep response: %r", maven_dep_response)
 
+        if not isinstance(maven_dep_response, MavenDependencyResult):
```

Comment:
I feel like we should probably error here or something right?

Reply:
See previous comment, I agree, but want a wholistic approach